app/lib/vendor_api_specification.rb

```diff
@@ -15,8 +18,46 @@ def as_hash
   end
 
   def spec
-    YAML
-      .load_file("config/vendor_api/v#{@version}.yml")
-      .deep_merge(YAML.load_file("config/vendor_api/experimental-v#{@version}.yml"))
+    (0..minor_version_number(@version)).each_with_object({}) do |minor_version, versions|
+      version_to_load = "#{major_version_number(@version)}.#{minor_version}"
+
+      if File.exist?(yaml_file_path(version_to_load))
+        yaml_for_version = YAML.load_file(yaml_file_path(version_to_load))
+
+        merge_experimental_spec!(yaml_for_version, version_to_load)
+      end
+
+      merge_draft_spec!(yaml_for_version)
```

Comment:
when testing locally and enabling v1.1, Im not sure why I'm getting the following `components/schemas/MultipleApplicationsResponse does not resolve to a valid object`, also I wonder if its ok merge two similar specs on top of each other? (draft then 1.1)

Reply:
@cwrw This seemed to be a corrupt spec file, the partial draft spec merged fine with v1.0.yml. After an hour or so of investigating I'm not sure what specifically was wrong so I overwrote v1.1 with the draft spec and cut out the pagination parts which seems to have fixed the issue. (Should have thought to diff the previous to current to find out what went wrong).
I've added a request spec for the `api-docs/reference` route for all versions and also tested `api-docs/draft`. This should help highlight similar failures of corrupt partial specs.